src/tests/e2etest/Tests.cs

```diff
@@ -0,0 +1,126 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using NUnit.Framework;
+using Microsoft.Azure.Devices.Shared;
+using System.Text.Json;
+using System;
+using System.Threading.Tasks;
+
+namespace e2etesting
+{
+    public class CommandRunnerTests : E2eTest
+    {
+        public enum Action
+        {
+            ActionNone = 0,
+            ActionReboot,
+            ActionShutdown,
+            ActionRunCommand,
+            ActionRefreshCommandStatus
+        }
+
+        public enum CommandState
+        {
+            Unknown = 0,
+            Running,
+            Succeeded,
+            Failed
+        }
+        public partial class CommandArguments
+        {
+            public string CommandId { get; set; }
+            public string Arguments { get; set; }
+            public Action Action { get; set; }
+            public int Timeout { get; set; }
+            public bool SingleLineTextResult { get; set; }
+        }
+
+        public partial class CommandStatus
+        {
+            public string CommandId { get; set; }
+            public long ResultCode { get; set; }
+            public long ExtendedResultCode { get; set; }
+            public string TextResult { get; set; }
+            public CommandState CurrentState { get; set; }
+        }
+
+        [Test]
+        public void CommandRunnerTest_ActionRunCommand_echo_HelloWorld()
```

Comment:
Lets keep the module tests in their own classes just as they were previously. This provides a model for other module developers to use.